src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.body;
+
+const succesMessage = document.createElement('div');
```

Comment:
Typo in the variable name 'succesMessage'. Consider renaming it to 'successMessage'.